collaborative-pre-merge-testing-for-multi-PR-features/README.md

```diff
@@ -0,0 +1,580 @@
+
+> **Professional Note:**
+> The automation workflows and templates provided here are based on Signadot's official patterns and best practices. They are designed to work when correct parameters, secrets, and valid container images are supplied. However, as with any CI/CD automation, users should validate these workflows in their own environment and adapt as needed for their specific use case and infrastructure.
+
+### 5.2. Required Templates
+
+- **Sandbox template:** `.signadot/sandbox-template.yaml`
+- **RouteGroup template:** `.signadot/routegroup-template.yaml`
+
+**.signadot/sandbox-template.yaml**
+
+```yaml
+name: "${name}"
+spec:
+  cluster: "${cluster}"
+  labels:
+    signadot/github-pull-request: "${signadot/github-pull-request}"
+    signadot/github-repo: "${signadot/github-repo}"
+    epic: "${epic}"
+  forks:
+    - forkOf:
+        kind: Deployment
+        namespace: hotrod
+        name: ${service}
+      customizations:
+        images:
+          - image: ${image}
+```
+
+**.signadot/routegroup-template.yaml**
+
+```yaml
+name: "${name}"
+spec:
+  cluster: "${cluster}"
+  match:
+    any:
+      - label:
+          key: epic
+          value: "${match_value}"
+  endpoints:
+    - name: hotrod-frontend
+      target: http://frontend.hotrod.svc:8080
+```
+
+### 5.3. Add Required GitHub Secrets
+
+In your repository settings, add the following secrets:
+
+- `SIGNADOT_API_KEY` (your Signadot API key)
+- `SIGNADOT_ORG` (your Signadot organization name)
+- `SIGNADOT_CLUSTER` (your Signadot cluster name)
+
+### 5.4. Workflow 1: Create Sandbox on PR
+
+Create `.github/workflows/create-pr-sandbox.yml`:
+
+```yaml
+name: Create PR Sandbox
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  create-sandbox:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # (Optional) Build and push your image
+      # - name: Build and push image
+      #   run: |
+      #     docker build -t your-registry/hotrod:${{ github.sha }} .
+      #     docker push your-registry/hotrod:${{ github.sha }}
+
+      - name: Install Signadot CLI
+        run: curl -sSLf https://raw.githubusercontent.com/signadot/cli/main/scripts/install.sh | sh
+
+      - name: Apply Sandbox
+        env:
+          SIGNADOT_API_KEY: ${{ secrets.SIGNADOT_API_KEY }}
+          SIGNADOT_ORG: ${{ secrets.SIGNADOT_ORG }}
+        run: |
+          signadot sandbox apply -f .signadot/sandbox-template.yaml \
+            --set cluster=${{ secrets.SIGNADOT_CLUSTER }} \
+            --set name="pr-${{ github.event.pull_request.number }}" \
+            --set image="your-registry/hotrod:${{ github.sha }}" \
+            --set labels."signadot/github-pull-request"="${{ github.event.pull_request.number }}" \
+            --set labels."signadot/github-repo"="${{ github.repository }}"
+```
+
+### 5.5. Workflow 2: Link Epic via PR Comment
```

Comment:
Would be good to show 1-2 screenshots of this github view showing users making the /epic comment and the result of this in the PR view.